### PR TITLE
chore: update black to 23.3.0

### PR DESCRIPTION
**Pull request by aarnphm**
Signed-off-by: Aaron <29749331+aarnphm@users.noreply.github.com>

## What does this PR address?

Fixes #(issue)

## Before submitting:

- [ ] Does the Pull Request follow [Conventional Commits specification](https://www.conventionalcommits.org/en/v1.0.0/#summary) naming? Here are [GitHub's
      guide](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/creating-a-pull-request) on how to create a pull request.
- [ ] Does the code follow BentoML's code style, both `make format` and `make lint` script have passed ([instructions](https://github.com/bentoml/BentoML/blob/main/DEVELOPMENT.md#style-check-auto-formatting-type-checking))?
- [ ] Did you read through [contribution guidelines](https://github.com/bentoml/BentoML/blob/main/CONTRIBUTING.md#ways-to-contribute) and follow [development guidelines](https://github.com/bentoml/BentoML/blob/main/DEVELOPMENT.md#start-developing)?
- [ ] Did your changes require updates to the documentation? Have you updated
      those accordingly? Here are [documentation guidelines](https://github.com/bentoml/BentoML/tree/main/docs) and [tips on writting docs](https://github.com/bentoml/BentoML/tree/main/docs#writing-documentation).
- [ ] Did you write tests to cover your changes?

### CODE REVIEW

File: examples/pytorch_mnist/pytorch_mnist_demo.ipynb [hunk 1]
```diff
@@ -221,7 +221,6 @@
     "\n",
     "    # K-fold Cross Validation model evaluation\n",
     "    for fold, (train_ids, test_ids) in enumerate(kfold.split(dataset)):\n",
-    "\n",
```
Comment (sauyon) [inline examples/pytorch_mnist/pytorch_mnist_demo.ipynb:224]: These don't look like formatter changes?

File: requirements/tests-requirements.txt [hunk 2]
```diff
@@ -19,4 +19,4 @@ protobuf<4.0dev
 grpcio
 grpcio-health-checking
 opentelemetry-instrumentation-grpc==0.35b0
-Pillow
\ No newline at end of file
+Pillow
```
Comment (sauyon) [inline requirements/tests-requirements.txt:22]: Is this relevant?

File: examples/pytorch_mnist/pytorch_mnist_demo.ipynb (hunk 1 above)
Comment (aarnphm) [reply examples/pytorch_mnist/pytorch_mnist_demo.ipynb:224]: that is from `black[jupyter]`

**Comment by aarnphm** [comment]
Closed this for #3927

**Pull request closed without merging**